api/swagger.yaml

```diff
@@ -0,0 +1,788 @@
+      - $ref: '#/components/schemas/ConstructCommand'
+      - $ref: '#/components/schemas/DecommissionCommand'
+      - $ref: '#/components/schemas/AttackCommand'
+      - $ref: '#/components/schemas/TradeCommand'
+      - $ref: '#/components/schemas/RenameCommand'
+    MoveCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - id
+        - where
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+          where:
+            type: integer
+            format: int64
+    StopCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - id
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+    ConstructCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - id
+        - where
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+          where:
+            type: integer
+            format: int64
+    DecommissionCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - id
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+    RenameCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - id
+        - name
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+          name:
+            type: string
+    AttackCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - id
+        - target
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+          target:
+            type: integer
+            format: int64
+    TradeCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - amount
+        - id
+        - resource
+        - where
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+          resource:
+            type: integer
+            format: int64
+          amount:
+            type: integer
+            format: int64
+          where:
+            type: integer
+            format: int64
+    Commands:
+      type: object
+      properties:
+        commands:
+          type: array
+          items:
+            $ref: '#/components/schemas/Command'
+    Credentials:
+      required:
+      - name
+      - password
+      - player
+      type: object
+      properties:
+        name:
+          type: string
+        password:
+          type: string
+        player:
+          type: string
+    Tick:
+      type: object
+      properties:
+        tick:
+          type: integer
+          description: Game tick in milliseconds
```

Comment:
not milliseconds. it is an index of the step of the simulation.

Reply:
This is obsolete (replaced by end-turn)